distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -280,6 +280,8 @@ type Number
 
     ## Checks equality of numbers, using an `epsilon` value.
 
+       Throws `Incomparable_Values` error if `Number.nan` is used as `self` or `that`.
```

Comment:
```suggestion
       ! Error Conditions
         If either of the arguments is `Number.nan`, an `Incomparable_Values` error is raised.
```


I don't want to impose a style too much, but I think this form may look a bit better.